lib/core/viewer/components/pdfViewer.component.scss

```diff
@@ -19,20 +19,33 @@
 
     &__toolbar {
         position: absolute;
-        bottom: 0;
+        bottom: 5px;
 
         left: 50%;
         transform: translateX(-50%);
 
+        .adf-toolbar .mat-toolbar {
+            max-height: 48px;
+            background-color: #ffffff;
```

Comment:
we need the same here +                background-color: mat-color($background, background);